UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiation.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
It seems you mistakenly copied this file to UnitsNet.Tests?

Reply:
True, didn't notice that